.github/workflows/container.yml

```diff
@@ -0,0 +1,39 @@
+name: Container Check
+
+on:
+  push:
+    paths-ignore:
+      - '**/*.md'
+  pull_request:
+    paths-ignore:
+      - '**/*.md'
```

Comment:
I think we should only call it if
- testdata/project-v4/Makefile
- testdata/project-v4/Dockerfile
- OR this file changes

Reply:
Agreed, will make change